### PR TITLE
add documentation and links.

### PR DESCRIPTION
**Pull request by vermasavita**
1. Documentation for all the components
2. Add links in the link buttons
3. Changed some styling.

### CODE REVIEW

File: components/badge/badge.html
Comment (ajitpsakri) [inline components/badge/badge.html:59]: You can remove these comments

**Review by ajitpsakri — APPROVED** [review]
There are some merge conflicts 
Apart from redundant comments, everything looks good

File: components/badge/badge.html
Comment (vermasavita) [reply components/badge/badge.html:59]: Hey, Ajit. I have to create those but not doing them right now because of deadline constraints. Just focusing bare minimum for now.

Ah, yes conflict is because I have to change few code lines I have already merged in my dev(main) branch. I'll resolve that during the merge.

Comment (vermasavita) [reply components/badge/badge.html:59]: Please approve if all good.

**Comment by Husain01** [comment]
There is a different color here being applied than your latest PR. Kindly check
Screenshot 2022-02-08 104937

**Comment by vermasavita** [comment]
@Husain01 yes because this is the new one. After merge this will be the final design.

**Review by Husain01 — APPROVED** [review]

**Pull request merged**
